lib/task-data/tasks/set-boot-disk.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015, EMC, Inc.
```

Comment:
Thanks. This was created sometime ago and I am doing the PR now. I will update it